locale/extract.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -o errexit
+
+DEB=$1
+OUT_TAR=$2
+
+ar -x "$DEB" data.tar.xz
+
+tar -xf data.tar.xz ./usr/lib/locale ./usr/share/doc/libc-bin/copyright
```

Comment:
To be conservative, should we do
```suggestion
tar -xf data.tar.xz ./usr/lib/locale/C.UTF-8 ./usr/share/doc/libc-bin/copyright
```
?